### DISCUSSION TITLE
BSD export templates by default?

### DISCUSSION THREAD

**Discussion by HappyGoFishing** [godotengine/godot-proposals#6434] [Platforms]
Is there any plan to include the BSD Operating Systems (OpenBSD, FreeBSD, etc) in the official export templates? unless there is some obscure reason (unknown to me) that prevents BSD templates being shipped, I don't understand why they cant be included by default.

If anyone knows more about BSD exports with Godot, please reply to this thread. 🙂
Thank you for listening to my discussion.

![image](https://user-images.githubusercontent.com/85656267/222959573-39b1e097-c412-404f-83aa-35beb63c76fa.png)

**Comment by AThousandShips** [comment]
Does the Linux export template not support it?

**Reply by HappyGoFishing** [reply]
If it does it is still makes no hint that it supports other UNIXes other than Linux.
perhaps having OpenBSD/X11 and FreeBSD/X11 would demystify it for developers.

**Comment by Calinou** [comment]
> unless there is some obscure reason (unknown to me) that prevents BSD templates being shipped, I don't understand why they cant be included by default.

While [Godot can be built for various *BSDs](https://github.com/Calinou/godot-ci-freebsd), building export templates for all *BSDs is difficult. We need to ensure these ports work correctly over time too, which is nontrivial given these have relatively few users. Remember that different *BSD distributions are **not** compatible with each other, unlike Linux (and there's no solution like Flatpak to avoid this).

**Comment by HappyGoFishing** [comment]
Thanks for clarifying. I guess that supporting the multiple BSDs is more complex than Linux, especially with their much smaller user base.
perhaps maybe just supporting Open and Free BSD would be fine, since they are the most used BSDs I'm pretty sure.
👋
